README.md

```diff
@@ -83,61 +56,40 @@ Use the following command to run the Busola with the [`core`](./core) and all ot
 npm run start
 ```
 
-To get the credentials required to access the local instance of the Kyma Busola at `http://busola-dev.kyma.local:4200`, follow the instructions from [this](https://kyma-project.io/docs/master/root/kyma#installation-install-kyma-on-a-cluster-access-the-cluster) document.
+After a while open the [http://localhost:4200](http://localhost:4200) address in your browser, then add your **init params** to the address to make it look like `http://localhost:4200?init=yourInitParams`.
```

Comment:
Should we provide link to enkode for init params

Reply:
@valentinvieriu

Reply:
Enkode is a temporary solution for now. But you are right @akucharska that we should mention it for now, but don't try to explain yet what are the fields